admin-node-setup.sh

```diff
@@ -29,5 +30,21 @@ if [ ! -f $manifest_dir/private.yaml ]; then
     exit -3
 fi
 
-cp -v $manifest_dir/public.yaml $kube_dir
-cp -v $manifest_dir/private.yaml $kube_dir
+tmp_dir=$(mktemp -d)
+
+cp $manifest_dir/public.yaml $tmp_dir
+cp $manifest_dir/private.yaml $tmp_dir
+
+for i in $(ls $images_dir/sles*.tag);do
+    metadata_file=$(basename $i .tag).metadata
+    image_name=$(cat $images_dir/$metadata_file | grep \"name\": | cut -d":" -f2 | cut -d\" -f2)
+    tag=$(cat $i)
+    echo "$0: Setting $image_name:$tag into public and private manifests"
+    sed -i -e "s%$image_name:__TAG__%$image_name:$tag%g" $tmp_dir/public.yaml
+    sed -i -e "s%$image_name:__TAG__%$image_name:$tag%g" $tmp_dir/private.yaml
+done
+echo "$0: Replacing public and private manifests"
+cp $tmp_dir/public.yaml $kube_dir
+cp $tmp_dir/private.yaml $kube_dir
```

Comment:
:) . It is a personal taste of mine ... I don't like moving files but prefer to copy and then remove them. I once lost all of my university assignments by moving files around...